pkg/apis/core/v1alpha1/types_schedulingprofile.go

```diff
@@ -46,15 +45,74 @@ type SchedulingProfileList struct {
 }
 
 type SchedulingProfileSpec struct {
-	// Plugins is the list of plugins to use when scheduling a resource
-	Plugins []SchedulingPlugin `json:"plugins"`
+	// Plugins specify the set of plugins that should be enabled or disabled.
+	// Enabled plugins are the ones that should be enabled in addition to the
+	// default plugins. Disabled plugins are any of the default plugins that
+	// should be disabled.
+	// When no enabled or disabled plugin is specified for an extension point,
+	// default plugins for that extension point will be used if there is any.
+	Plugins *Plugins `json:"plugins"`
```

Comment:
Should this field be `omitempty` and `+optional`? In case we only want to customise the plugin config

Reply:
yes i think so, overlooked this part

Reply:
fixed